.github/workflows/packaging.yml

```diff
@@ -303,6 +303,7 @@ jobs:
           - almalinux-8
           - almalinux-9
           - debian-bookworm
+          - debian-bookworm-arm64
```

Comment:
We want to use `ubuntu-24.04-arm` for arm64 build for performance. QEMU is slow.